environments/dev/hiera/common.json

```diff
@@ -0,0 +1,68 @@
+{
+    "classes": [
+        "::localconfig::ordering",
+        "::localconfig::hostnames",
+        "::ghservice::deps::common",
+        "::ghservice::apache",
+        "::ghservice::grasshopper",
+        "::ghservice::postgresql",
+        "::ghservice::ui"
+    ],
+
+    /* Domains and hostnames */
+    "admin_hostname": "admin.%{hiera('tenant_hostname')}",
+    "tenant_hostname": "ec2-nn-nn-nn-nn.eu-west-1.compute.amazonaws.com",
```

Comment:
Could this be pulled in by puppet? Is there a system variable we could speak to?